ledger/participant-state/kvutils/BUILD.bazel

```diff
@@ -68,6 +68,7 @@ da_scala_library(
         "//daml-lf/archive:daml_lf_dev_archive_java_proto",
         "//daml-lf/data",
         "//daml-lf/transaction",
+        "//daml-lf/transaction-scalacheck",
```

Comment:
Shouldn't this rather be `transaction-tests-lib` or similar? I.e., it seems to contain classes to support testing in general.

Reply:
you are right, it is now more general than scalacheck. However I am not sure it is worthwhile to rename the libraries the directory, and the maven coordinate.

i am not sure it is worthwhile to create a news library neither.

what do you think?

cc @S11001001

Reply:
I think it's worth renaming the above as it will ensure less confusion. However, it'd probably make sense to do these renames in a separate PR.

Reply:
I don't think it's worth renaming, but sure, go for it 🤷‍♂️